api/envoy/api/v2/endpoint.proto

```diff
@@ -114,4 +115,28 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   Policy policy = 4;
+
+  // References endpoint groups by name.
+  repeated Egds endpoint_groups = 6;
+}
+
+// New xDS payload. The size of endpoints is fully arbitrary.
+message EndpointGroup {
+  string name = 1 [(validate.rules).string = {min_bytes: 1}];
+
+  // List of endpoints to load balance to.
+  repeated endpoint.LocalityLbEndpoints endpoints = 2;
+
+  // Map of named endpoints that can be referenced in LocalityLbEndpoints.
+  // [#not-implemented-hide:]
+  map<string, endpoint.Endpoint> named_endpoints = 3;
```

Comment:
This field is not currently in use, but I think it should be retained because the endpoint group just groups the endpoints and the original definition should not be changed.

Reply:
I think you mean https://developers.google.com/protocol-buffers/docs/proto3#oneof